### DISCUSSION TITLE
More than 3 areas ?

### DISCUSSION THREAD

**Discussion by Strega-Technologies** [mikey0000/Mammotion-HA#144] [General]
is it possible to extend the areas to 10, please ? I have a Luba-1 5000 and use all 10x areas. Thanks in advance.

**Comment by mikey0000** [comment]
Should be working on latest version, every so often hit sync

**Comment by Strega-Technologies** [comment]
Thank you for your answer. Pressing on the button provides me with 4x areas only, one of which is already present as area 1=
![areas](https://github.com/user-attachments/assets/87f1defe-3903-403f-abd0-2453e24f1c8f)
thank you.

**Reply by mikey0000** [reply]
Yup so just repeat till they all show up, I know it's not ideal and I will get to improving this.

**Comment by Strega-Technologies** [comment]
Yes, working now ! Once fully finished, your integration will be extremely powerful with HA offering endless mowing possibilities... Thank you for your great work !
